### ISSUE TITLE
 Improve the plotting and exporting facilities

### ISSUE THREAD

**Issue by spedygiorgio** [spedygiorgio/markovchain#20]
Maybe try to provide an option to export in Rgrapviz format or to import from igraph.

**Comment by spedygiorgio** [comment]
Also have a look to diagram R package (http://cran.r-project.org/web/packages/diagram/vignettes/diagram.pdf) There is an example to plot a transitgion matrix. As for Rgraphviz, I imagine a hidden functrion that receives the markovchain objcte in input and  then return the plotted object. Then we will decide on how to integrate in the package. Write them i an seprarate R code (No Rcpp needed)

**Comment by spedygiorgio** [comment]
Also have a look to the DiagrammeR http://cran.r-project.org/web/packages/DiagrammeR/vignettes/DiagrammeR.html package

**Comment by tskang** [comment]
It is done. The following functions were implemented. 
- .plotdiagram(object, ...)
- .plotDiagrammeR(object)

**Comment by spedygiorgio** [comment]
Hi, good starting work. I have reworked on (see my commit). To close the issue we need:
1. a) DiagrammerR ways shows number. We need the states names in. Also we need to pass supplementary parameters to control the graph (...)
2. I have reworked the plot method. I am not sure it will properly pass all CRAN checks. So have a look on how to document S4 classes ( see my post on http://stackoverflow.com/questions/31366088/rewriting-standard-s4-methods) . Amend the documentation (man).
3. Update my main vignette with two examples, one using diagram, the latter using DiagrammeR
